app/models/flexible_criterion.rb

```diff
@@ -3,7 +3,8 @@
 # Represents a flexible criterion used to mark an assignment that
 # has the marking_scheme_type attribute set to 'flexible'.
 class FlexibleCriterion < Criterion
-
+  extend ActionView::Helpers
```

Comment:
Actually, please revert the internationalization changes. You can do it separately later.